spec/foundation_rails_helper/flash_helper_spec.rb

```diff
@@ -42,13 +39,16 @@
 
   it "displays flash message with overridden key matching" do
     allow(self).to receive(:flash).and_return("notice" => "Flash message")
-    node = Capybara.string display_flash_messages(key_matching: { notice: :alert })
+    node =
+      Capybara.string display_flash_messages(key_matching: { notice: :alert })
     expect(node).to have_css("div.callout.alert", text: "Flash message")
   end
 
   it "displays flash message with custom key matching" do
     allow(self).to receive(:flash).and_return("custom_type" => "Flash message")
-    node = Capybara.string display_flash_messages(key_matching: { custom_type: :custom_class })
+    node = Capybara.string(
```

Comment:
This is an excellent point and I totally agree that I need to figure out a consistent way to separate one line of code onto multiple lines.  I just haven't found the best approach, yet.

This is the order of my current process takes for assignments:

1. Try putting the second expression onto the next line
2. Try to align attributes
3. Add parenthesis to the outside method and move it's parameters to separate lines.
4. Repeat 3 until our line is short enough

For the line in question, skipping to 3 is the only path I see.

When I started to be stricter on line length, I would start at 3.  However, I noticed that 2 is very common in Ruby, and have grown to be accustomed to it.

I have not found any documentation on how to best approach this problem, but would appreciate feedback or a good resource.